relay_rpc/src/auth/cacao/signature/test_helpers.rs

```diff
@@ -0,0 +1,79 @@
+use {
+    super::eip191::eip191_bytes,
+    alloy_node_bindings::{Anvil, AnvilInstance},
+    alloy_primitives::Address,
+    k256::ecdsa::SigningKey,
+    regex::Regex,
+    sha2::Digest,
+    sha3::Keccak256,
+    std::process::Stdio,
+    tokio::process::Command,
+    url::Url,
+};
+
+pub async fn spawn_anvil() -> (AnvilInstance, Url, SigningKey) {
+    let anvil = Anvil::new().spawn();
+    let provider = anvil.endpoint().parse().unwrap();
+    let private_key = anvil.keys().first().unwrap().clone();
+    (
+        anvil,
+        provider,
+        SigningKey::from_bytes(&private_key.to_bytes()).unwrap(),
+    )
+}
+
+pub async fn deploy_contract(rpc_url: &Url, private_key: &SigningKey) -> Address {
+    let key_encoded = data_encoding::HEXLOWER_PERMISSIVE.encode(&private_key.to_bytes());
+    let output = Command::new("forge")
+        .args([
+            "create",
+            "--contracts=contracts",
+            "Eip1271Mock",
+            "--rpc-url",
+            rpc_url.as_str(),
+            "--private-key",
+            &key_encoded,
+            "--cache-path",
+            "target/.forge/cache",
+            "--out",
+            "target/.forge/out",
+        ])
+        .stdout(Stdio::piped())
+        .stderr(Stdio::piped())
+        .spawn()
+        .unwrap()
+        .wait_with_output()
+        .await
+        .unwrap();
+    println!("forge status: {:?}", output.status);
```

Comment:
were these prints intended to be there?

Reply:
Yeah these are helpful if this tool fails to start for some reason. They don't get printed unless the test fails.